<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="leftView" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Journal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newEntryButton">
         <property name="text">
          <string>New Entry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lockButton">
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox">
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightView" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="dateView">
            <property name="text">
             <string>Empty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="moodSelection">
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Happy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sad</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Excited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Calm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Neutral</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Idk</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleView">
         <property name="placeholderText">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="contentView">
         <property name="placeholderText">
          <string>Type something here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>49</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
